conf/deployment/ibmcloud/ipi_3az_rhcos_3m_3w.yaml

```diff
@@ -5,17 +5,17 @@ DEPLOYMENT:
 ENV_DATA:
   platform: 'ibm_cloud'
   deployment_type: 'ipi'
-  region: 'us-south'
+  region: 'us-east'
   base_domain: 'ibmcloud2.qe.rh-ocs.com'
   worker_instance_type: 'bx2-16x64'
   worker_availability_zones:
-    - 'us-south-1'
-    - 'us-south-2'
-    - 'us-south-3'
+    - 'us-east-1'
```

Comment:
Will this have effect on AWS cleanup job? Do we do cleanup only on one region? Please check. Maybe @clacroix12 knows?

Reply:
Doesn't hold up my approval but I was wondering why we are changing availability zones. Is it just to keep it in line with our AWS deployments?